third_party/CMakeLists.txt

```
@@ -1,9 +1,15 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 
 project(torchaudio_third_parties)
-include(ExternalProject)
 
-set(INSTALL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/install)
+option(BUILD_SOX "Build libsox statically")
+option(BUILD_TRANSDUCER "Build transducer")
```

nit: "Build transducer statically" to echo libsox?
```suggestion
option(BUILD_TRANSDUCER "Build transducer statically")
```

---

`BUILD_SOX` has a slightly different connotation, as (ignoring Windows case) sox is bound anyway, but with `BUILD_SOX` it is binding the statically built one.

While `BUILD_TRANSDUCER` switch on/off the both build and bind, so there is no expected case where transducer library is dynamically bound.

But I guess it does not harm to be more precise.